### PR TITLE
[15.0][ADD]invoice_export

### PR DESCRIPTION
**Pull request by dsolanki-initos**
@jans23 FYI

### CODE REVIEW

**Comment by dsolanki-initos** [comment]
@yajo Can you please review this PR?

**Comment by dsolanki-initos** [comment]
@rafaelbn Can you please review this PR?

**Comment by dsolanki-initos** [comment]
@HaraldPanten can you please review this PR?

**Comment by rafaelbn** [comment]
Hello @dsolanki-initos 

- this can be done without any module.
- you are exporting sales, not invoices, no sense this module in this repo

I recommend you to check https://www.youtube.com/watch?v=fYOFRk4P0PQ

About pinging everyone without to much sense you could have contradictory effect.

I like new contributors in OCA, I'm happy to see you here. Just think in both sides.

Good luck and see you soon! 😄

**Comment by dsolanki-initos** [comment]
@mariadforgeflow Could you please review this PR?

**Review by ikapasi-initos — APPROVED** [review]
Code review and functional test
LGTM

File: invoice_export/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Invoice Export",
+    "version": "15.0.1.0.0",
+    "depends": ["account"],
+    "data": [
+        "security/ir.model.access.csv",
+        "wizard/invoice_export_excel_view.xml",
+    ],
+    "author": "Nitrokey GmbH, Odoo Community Association (OCA)",
```
Comment (AungKoKoLin1997) [inline invoice_export/__manifest__.py:9]:
```suggestion
    "author": "Nitrokey GmbH, Odoo Community Association (OCA)",
    "category": "Accounts",
```

**Review by AungKoKoLin1997 — CHANGES_REQUESTED** [review]
Can you please move Invoice Excel from sale to invoicing app. That is the invoice export, not sale export. So, I believe that menu should be in invoicing app. And please also add category in minifest file

**Comment by dsolanki-initos** [comment]
> Can you please move Invoice Excel from sale to invoicing app. That is the invoice export, not sale export. So, I believe that menu should be in invoicing app. And please also add category in minifest file

@AungKoKoLin1997 I've modified as per your changes could can you please review it?

File: invoice_export/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+To use this module, you need to:
+
+Go to Sales -> Orders -> Invoice Excel
+
```
Comment (AungKoKoLin1997) [inline invoice_export/readme/USAGE.rst]: Go  to Invoicing -> Reporting -> Invoice Excel

**Review by AungKoKoLin1997 — COMMENTED** [review]
Can you also please regenerate README.rst?

File: invoice_export/readme/USAGE.rst (hunk 2 above)
Comment (dsolanki-initos) [reply invoice_export/readme/USAGE.rst]: @AungKoKoLin1997 Yes will add it

Comment (dsolanki-initos) [reply invoice_export/readme/USAGE.rst]: @AungKoKoLin1997 I've made changes, Can you please review it?

**Pull request closed without merging**